CODING.md

```diff
@@ -31,6 +31,16 @@ gulp
 ## Guidelines for `custom-civicrm.css`
 Any style changes that are aimed at making the core screens look like they are part of the Bootstrap theme, should go here.
 
+### Disable stylelint rules
+Any .scss file under the `civicrm/` folder must have this annotation at the very top
+
+```scss
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
```

Comment:
Could we just set a reasonable amount of `max-nesting-depth` at https://github.com/compucorp/org.civicrm.shoreditch/pull/88/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R33 so we don't pollute files too much? You can set it to 999. While it may slow things down a bit, I don't see it as an issue since you build files not that often. Same for `selector-max-compound-selectors` and `selector-max-id`.

As per `selector-no-qualifying-type` I think you can ignore it by `ignore: ["attribute", "class", "id"]`
https://stylelint.io/user-guide/rules/selector-no-qualifying-type/

Reply:
we can't do that, because those configuration will be applied to `bootstrap/` as well, and i'm not willing to lower the bar for the bootstrap side of civicrm because of `civicrm/`


> As per selector-no-qualifying-type I think you can ignore it by ignore: ["attribute", "class", "id"]

for civicrm/ we would have to ignore all of the cases, so we just disable the rule (also, i don't think you can do anything more than enabling/disabling rules in the annotations)